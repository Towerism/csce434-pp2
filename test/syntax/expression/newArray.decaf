int asdf() {
    NewArray(qwer = 5, AASSDDFF[]);
}