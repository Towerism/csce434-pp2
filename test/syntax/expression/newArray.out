
   Program: 
  1   FnDecl: 
         (return type) Type: int
  1      Identifier: asdf
         (body) StmtBlock: 
  2         NewArrayExpr: 
  2            AssignExpr: 
  2               FieldAccess: 
  2                  Identifier: qwer
  2               Operator: =
  2               IntConstant: 5
  2            ArrayType: 
  2               NamedType: 
  2                  Identifier: AASSDDFF
